index.html

```diff
@@ -9,24 +9,24 @@
     <title>Aziz Mejri - Portfolio</title>
   </head>
   <body>
-    <header>
+    <div id="header">
       <h1 id="logo">Aziz Mejri</h1>
       <button type="button" class="menu-bars">
         <i class="fas fa-bars"></i>
       </button>
-      <nav id="desktop-nav">
+      <div id="desktop-nav">
         <ul>
-          <li><a class="nav-link" href="#projects">Portoflio</a></li>
-          <li><a class="nav-link" href="#about-me">About</a></li>
+          <li><a class="nav-link" href="#">Portoflio</a></li>
```

Comment:
- [ ]  kindly make sure that when the user tap any of the options mobile menu. as the previous code reviewer mentioned that it should redirect to the specific section on the page. please check the video I showed the bug.
https://www.loom.com/share/923543a959434d6aad239234466ee50f